zboss/CHANGELOG.rst

```diff
@@ -9,6 +9,20 @@ Changelog
 
 All notable changes to this project in the |NCS| are documented in this file.
 
+nRF Connect SDK v2.3.0
```

Comment:
@greg-fer Is this ok to mark this already as NCS release instead of `Main branch`?

Reply:
That is ok.

Reply:
Leaving this up to @b-gent and @shantha-14.

Reply:
On second thought, let's change to Main branch, and update this in the version update PR. FYI, @b-gent

Reply:
yes, let's do it this way